packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -149,9 +128,14 @@ function toTimestamp(date: number | Date) {
   return date;
 }
 
-function isDailyTriggerInput(
-  trigger: WeeklyTriggerInput | DailyTriggerInput | CalendarTriggerInput | TimeIntervalTriggerInput
-): trigger is DailyTriggerInput {
+type TriggerInput =
+  | YearlyTriggerInput
+  | WeeklyTriggerInput
+  | DailyTriggerInput
+  | CalendarTriggerInput
+  | TimeIntervalTriggerInput;
```

Comment:
why don't we use `SchedulableNotificationTriggerInput` from Notifications.types? Would need to handle `DateTriggerInput` properly though.